Fix  FuseSoC config generator

joannabrozek opened the pull request:
After the following change:
https://github.com/chipsalliance/Cores-SweRV/commit/ec254f5#diff-7682db868261ad0b18c9d5ca090189ad3eac89bbbcbc44530c9b2a40ee71a6eaL37
the FuseSoC tool started failing due to a missing directory
and this commit fixes it.

agrobman commented:
Joanna, the swerv.config generates files in current directory now: $PWD/snapshots/<snapshot_name> by default.
(Not in $RV_ROOT/configs/snapshots/..).

joannabrozek committed:
Fix the build path for FuseSoC

After the following change:
https://github.com/chipsalliance/Cores-SweRV/commit/ec254f5#
diff-7682db868261ad0b18c9d5ca090189ad3eac89bbbcbc44530c9b2a40ee71a6eaL37
the fusesoc tool started failing due to a missing directory
and this commit fixes it.

joannabrozek commented:
@agrobman, I changed the directory according to the default build directory as you said. I just want to fix FuseSoC tool to get dependency list.

olofk commented:
Thanks for reporting. I see what happened now. When the code dump for version 1.9 was commited it overwrote swerv_config_gen.py with an older version. I think restoring the version from before the 1.9 release is the proper fix.

olofk commented:
To be explicit, this is the version we want https://github.com/chipsalliance/Cores-SweRV/blob/7332edc0adaa7e9a0c842d169154429e8d987786/configs/swerv_config_gen.py

aprnath commented:
Thanks Olof. I have reverted to that commit (0242c9e). Please confirm.

olofk commented:
Yes! That is correct. Thanks. If this works for @joannabrozek  too I think we can close the issue

aprnath commented:
Thanks Olof.
Hi Joanna, please use this latest commit and see if it works for you.

joannabrozek commented:
Yes, It works. Thanks a lot for help.
